shared/constants/countries.ts

```diff
@@ -0,0 +1,256 @@
+export enum Country {
+  AFGHANISTAN = 'AFGHANISTAN',
```

Comment:
nit: keys should be SentenceCase, (values can remain as is)

e.g.
```ts
export enum Country {
  Afghanistan = 'AFGHANISTAN',
  ...
}
```

PS: Probably don't need to manually do it, can probably do it in a javascript/node playground haha, by 
1. converting the enum into a `object`, 
2. then running some sort of key-change function on the object, then 
3. JSONify the object,
4. copy paste back into this enum

Reply:
Addressed by the latest commit `5d1da632a04966ad5c89153086406a78bca4f807`